pkg/kp/healthmanager.go

```diff
@@ -94,6 +95,24 @@ const (
 	// parmeter controls how long it takes for clients to notice that health checks have
 	// stopped.
 	SessionTTL = 15 * time.Second
+
+	// MaxWritesPerMinute sets the average number of writes per minute per service
+	// that will be sent to Consul to update health.
+	MaxWritesPerMinute = 1
```

Comment:
Maybe. The uncertainty here is why I want to make it configurable.  We can see what happens.

But I don't mind making the built-in default 4.